lab-james/lib/use-call.js

```diff
@@ -0,0 +1,19 @@
+'use strict';
+
+module.exports = exports = {};
+
+exports.Game = function(title, genre) {
+  this.title = title;
+  this.genre = genre;
+};
+
+exports.PcGame = function(title, genre) {
+  Game.call(this, title, genre);
```

Comment:
Nice use of call. Apply will work similiar to what you have here, but you need to pass in an array of arguments instead of comma separated values.